### ISSUE TITLE
[Experimental] Exception when closing the game

### ISSUE THREAD

**Issue by broken-usb** [mattymatty97/LTC_AsyncLoggers#9]
AsyncLoggers is always printing this exception on the console/log file whenever i close the game:

```
[Error  :AsyncLoggers] Exception writing log to db [Info LethalPerformance]: System.Threading.ThreadAbortException: Thread was being aborted.
  at (wrapper managed-to-native) SQLite.SQLite3.Step(intptr)
  at SQLite.PreparedSqlLiteInsertCommand.ExecuteNonQuery (System.Object[] source) [0x000c8] in ./src/Sqlite/SQLite.cs:3725 
  at SQLite.SQLiteConnection.Insert (System.Object obj, System.String extra, System.Type objType) [0x000c4] in ./src/Sqlite/SQLite.cs:1854 
  at SQLite.SQLiteConnection.Insert (System.Object obj) [0x00005] in ./src/Sqlite/SQLite.cs:1720 
  at AsyncLoggers.BepInExListeners.SqliteListener.LogEvent (System.Object sender, BepInEx.Logging.LogEventArgs eventArgs) [0x00093] in ./src/BepInExListeners/SqliteListener.cs:38 
[Error  :AsyncLoggers] Exception dispatching log to SqliteListener: System.Threading.ThreadAbortException: Thread was being aborted.
  at AsyncLoggers.BepInExListeners.SqliteListener.LogEvent (System.Object sender, BepInEx.Logging.LogEventArgs eventArgs) [0x000cb] in ./sr
```

Possible incompatibility with [Lethal Performance](https://thunderstore.io/c/lethal-company/p/DiFFoZ/LethalPerformance/) since it mentions it on the log?

**Comment by broken-usb** [comment]
With version 2.1.1(the most recent one) this changed from a exception when closing to a "crash" (it will display the Unity crash thingy when you close the game) when closing the game, from what i tested it happens both on Lethal Company and Risk of Rain 2.

**Comment by broken-usb** [comment]
Crash got fixed on 2.1.2 but the Exception is still present

**Comment by mattymatty97** [comment]
nothing much i can do if mods try to write logs after the game ( and more importantly BepInEx ) has closed
